From 24f652a1a53ba97c70fa8bb22bf45fa028ec8b24 Mon Sep 17 00:00:00 2001
From: Heinrich Toews <heinrich.toews@wago.com>
Date: Thu, 18 Apr 2019 14:51:47 +0200
Subject: [PATCH] defaultenv-pfc: config: set partitions binding to donttouch

The new AT24 driver in kernel 5.0.7 has nvmem support included to
add partitons to the device. The barebox partition fixup is causing an error
while probing the new driver. So we just disable fixing up the partition node.

Signed-off-by: Heinrich Toews <heinrich.toews@wago.com>
---
 defaultenv/defaultenv-pfc-base/config | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/defaultenv/defaultenv-pfc-base/config b/defaultenv/defaultenv-pfc-base/config
index 16d95d9..1bb7ec7 100644
--- a/defaultenv/defaultenv-pfc-base/config
+++ b/defaultenv/defaultenv-pfc-base/config
@@ -192,3 +192,5 @@ magicvar -a global.linux.bootargs.watchdog "board specific watchdog kernel param
 [ -z "${global.bootchooser.retry}" ] && global.bootchooser.retry=1
 
 [ -z "${global.linux.bootargs.watchdog}" ] && global.linux.bootargs.watchdog="omap_wdt.early_enable omap_wdt.timer_margin=${global.pfc.watchdog_timeout_linux_boot}"
+
+[ "${global.of_partition_binding}" != "donttouch" ] &&	global.of_partition_binding=donttouch
-- 
2.7.4

